src/js/components/DataTable/ExpanderCell.js

```diff
@@ -35,11 +42,17 @@ const ExpanderControl = ({ context, expanded, onToggle, pad, ...rest }) => {
   );
 
   if (onToggle) {
+    let a11yTitle;
```

Comment:
A logic suggestion:

```
let a11yTitle = expanded ? 'collapse' : 'expand';
if (rowExpandedLabel) a11yTitle = `${a11yTitle} ${rowExpandLabel}`;
```

Reply:
Should we internationalize the `expand` and `collapse` strings?

Reply:
we can? I think it would help forsure!

Reply:
implemented logic suggestion and internationalized the expand and collapse